backend/app/main.py

```diff
@@ -141,6 +141,22 @@ def get_developer(
     return [app for app in result if app]
 
 
+@app.get("/eol/rebase")
+def get_eol_rebase():
+    return db.get_json_key("eol_rebase")
+
+
+@app.get("/eol/rebase/{appid}")
+def get_eol_rebase_appid(
+    appid: str,
+):
+    for new_id, old_id_list in db.get_json_key("eol_rebase").items():
```

Comment:
This is exactly what I wanted to get rid of, you've now moved all work from the database to the backend. Which basically leads to loading it completely into memory and scanning it.

Reply:
Should be fixed with the latest commit